src/vm/database/clarity_db.rs

```diff
@@ -24,6 +25,13 @@ use vm::costs::CostOverflowingMath;
 
 const SIMMED_BLOCK_TIME: u64 = 10 * 60; // 10 min
 
+// not sure if this should actually only happen in developer-mode,
```

Comment:
I'm inclined to keep this on by default, and only turn it off for people who are actually affected by it.

Reply:
Yep, works for me

Reply:
Removed conditional build flag in 748a98b40ee2c0bd53272e5e468bf180074573b8